internal/app/go-filecoin/retrieval_market_connector/retrieval_api_test.go

```diff
@@ -0,0 +1,205 @@
+package retrieval_market_connector_test
```

Comment:
GFC prefers `fake` to `test` for implementations only used in tests. GFC convention is to put this in a file called `testing`.